### ISSUE TITLE
[packaging] README file is not in rst file format

### ISSUE THREAD

**Issue by mbukatov** [Tendrl/node_agent#90]
We should convert README file from ascii doc file format into [ReStructuredText](https://en.wikipedia.org/wiki/ReStructuredText). This can be done quite easily with [pandoc](https://apps.fedoraproject.org/packages/pandoc).

## Reasoning

It's a common practice to use [ReStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) file format for [README file in python projects](https://packaging.python.org/distributing/?highlight=readme#readme-rst), which is also commonly used for documentation of python projects (see sphinx). Moreover PyPI can only handle rst file format.

Also note that we already have sphinx documentation in tendrl-node-agent project, so having README in ascii doc seems weird and [prevents easy migration of text from README file into sphinx managed documentation](https://github.com/Tendrl/node_agent/issues/89).

**Comment by mbukatov** [comment]
I was wrong: [pandoc doesn't support converting from asciidoc files](https://github.com/jgm/pandoc/tree/8ced8cbc6e03212f7a9bd871d74ba0e2622c6233/src/Text/Pandoc/Readers).

**Comment by mbukatov** [comment]
That said, conversion was quite straightforward even without pandoc asciidoc reader. See the pull request.

**Comment by r0h4n** [comment]
If PyPi/python requires .rst docs we will have them. But we should not delete .adoc docs, there's no doubt that adoc is much better. And asciidoctor is supported on github

Please change the PR and keep both .rst and .adoc files

**Comment by mbukatov** [comment]
@r0h4n but that would create maintenance burden: who would keep those files in sync?

**Comment by mbukatov** [comment]
> If PyPi/python requires .rst docs we will have them.

It's not a hard requirement, but rather widely used practice (as described in link I referenced in description of this issue). This is amplified by the fact that recommended and most popular documentation management tools used in python projects are based on rst tools and sphinx. What actually *has to be* in rst syntax is a value for [long_description option insetup.py](https://packaging.python.org/distributing/?highlight=readme#description). Only when a python projects uses the README file as a value for`long_description`, it makes the rst syntax for README file mandatory.

In our case we have a `doc` directory in the repository, which contains sphinx based documentation. And I don't see a value in having README file in one format (which is going against common  python practices) and the documentation in another. I noticed that when I realized we would need to do a syntax conversion to fix https://github.com/Tendrl/node_agent/issues/89 This issue and a pull request was done to make fix for #89 easier. Also I don't like to go against common python practices without a good reason.
